data/defaults.yaml

```diff
@@ -312,3 +312,9 @@ prometheus::grok_exporter::package_ensure: 'latest'
 prometheus::grok_exporter::package_name: 'grok_exporter'
 prometheus::grok_exporter::user: 'grok-exporter'
 prometheus::grok_exporter::version: '1.0.0.RC4'
+prometheus::jmx_exporter::extra_groups: []
```

Comment:
please add the data to the exporter class directly, not into defaults.yaml. That's a legacy leftover we need to cleanup.

Reply:
Done